spec/features/search_spec.rb

```diff
@@ -36,23 +82,57 @@
       expect(page).to have_content('1 result for "practice"')
 
       # show practices that are approved/published
-      @user_practice2.update(published: true, approved: true)
-      visit '/search'
-
+      @user_practice2.update_attributes(published: true, approved: true)
+      update_practice(@user_practice)
+      visit_search_page
       fill_in('practice-search-field', with: 'practice')
-      click_button('Search')
+      search
 
       expect(page).to be_accessible.according_to :wcag2a, :section508
-
       expect(page).to have_content(@user_practice.name)
       expect(page).to have_content(@user_practice2.name)
       expect(page).to have_content('2 results for "practice"')
 
       # test facility data map for name, positive case
       expect(page).to have_content('Yakima VA Clinic')
+    end
+  end
+
+  describe 'Cache' do
+    it 'Should be reset if certain practice attributes have been updated' do
+      add_search_to_cache
 
+      expect(cache_keys).to include("searchable_practices")
+
+      update_practice(@user_practice)
+
+      expect(cache_keys).to include("searchable_practices")
     end
 
+    it 'Should be reset if a new practice is created through the admin panel' do
+      add_search_to_cache
+
+      expect(cache_keys).to include("searchable_practices")
+
+      login_as(@admin, :scope => :user, :run_callbacks => false)
+      visit '/admin'
+      click_link('Practices')
+      click_link('New Practice')
+      fill_in('Practice name', with: 'The Newest Practice')
+      fill_in('User email', with: 'practice_owner@va.gov')
+      click_button('Create Practice')
+      latest_practice = Practice.last
+
+      visit '/search?=newest'
+      expect(page).to_not have_content(latest_practice.name)
+
+      publish_practice(latest_practice)
```

Comment:
the tests kept failing both locally and in CI after the publish... 
`find('h1#overview', text: latest_practice.name)`
adding this hack-y wait helped the tests to pass... 😅 let me know if you have a more elegant alternative 😄

Reply:
thanks! let's hack away :axe: